src/PIL/ImageMode.py

```diff
@@ -23,18 +23,25 @@
 class ModeDescriptor:
     """Wrapper for mode strings."""
 
-    def __init__(self, mode, bands, basemode, basetype, typestr):
+    def __init__(
+        self,
+        mode: str,
+        bands: str | tuple[str, ...],
```

Comment:
```suggestion
        bands: tuple[str, ...],
```
I don't think `bands` is ever a string.